test-clients/src/main/resource/contract/solidity/hip-206/HederaTokenService.sol

```diff
@@ -386,16 +386,6 @@ abstract contract HederaTokenService is HederaResponseCodes {
         (responseCode) = success ? abi.decode(result, (int32)) : HederaResponseCodes.UNKNOWN;
     }
 
-    /// Operation to unpause token
-    /// @param token The token address to be unpaused
-    /// @return responseCode The response code for the status of the request. SUCCESS is 22.
-    function unpauseToken(address token) external returns (int responseCode)
```

Comment:
Is this removed intentionally ?

Reply:
Yep; a proper local Solidity dev setup wouldn't be using these files in any form, so I just needed to delete this to compile that test case.